### ISSUE TITLE
automatically generated Github Webhook Integration: response:400

### ISSUE THREAD

**Issue by Nic30** [rtfd/readthedocs.org#5426]
## Details

* Read the Docs project URL: https://netlistdb.readthedocs.io/
* Build URL (if applicable):
* Read the Docs username (if applicable): nic30

## Expected Result

*A description of what you wanted to happen*
I added the webhook as described on the https://docs.readthedocs.io/en/latest/webhooks.html
I used the webhook url https://readthedocs.org/api/v2/webhook/netlistdb/78249/ to Github repo.
Now I would like the push to actually trigger the doc build.

The Github repository is in organization. The readthedocs account is my personal.

## Actual Result

*A description of what actually happened*
The request arrives to integration in readthedocs and I can see it in the integration page.
However it complains about signature. (I let signature empty as described.)
```
Status:
    400
{
  "detail": "Payload not valid, invalid or missing signature"
}
```

Also the problem is that the error msg. is too generic.

**Comment by Nic30** [comment]
However I just deleted the webhook and added new and it is working.
(The first was generated by "new project" wizard. Maybe it does not work correctly)

**Comment by brannondorsey** [comment]
I too had this same problem. Removing the autogenerated hook and replacing it with my own (following [these instructions](https://docs.readthedocs.io/en/stable/webhooks.html#webhook-creation)) fixed this problem.

**Comment by Nic30** [comment]
@brannondorsey Did also the automatic configuration created the wrong hook?

If this is a case there is a bug which affect every new user and it kind of mean as everything seems to be configured properly and it just does not work.
Can you confirm this?

**Comment by brannondorsey** [comment]
I can confirm that the automatic configuration created an invalid hook. Editing this hook didn't even work, I had to add a completely new hook in order to trigger builds correctly.

**Comment by Nic30** [comment]
Yes I had to do the same thing. It seems that there is really the problem with the automatically generated github webhook and everything seems to have right values however it does not work.

**Comment by MaisaMilena** [comment]
Creating a new webhook did not work for me. I figure out that in my case the Payload URL that was being generated (i.e readthedocs.org/api/v2/webhook/**my-lang**/96993/) was using the old name of my repository. I had recently changed my GitHub project's name from "my-lang" to "my-new-lang". So, to solve this I deleted the project and created it again.
- This [issue](https://github.com/readthedocs/readthedocs.org/issues/2677) helped me
- The automatically generated github webhook works now
